src/queryCache.js

```diff
@@ -230,9 +231,7 @@ export function makeQueryCache() {
         return
       }
       query.staleTimeout = setTimeout(() => {
-        if (query) {
-          dispatch({ type: actionMarkStale })
-        }
+        dispatch({ type: actionMarkStale })
```

Comment:
I couldn't see anywhere in the code where `query` would be `falsy` and especially now that we're clearing the timeout when they're removed I think we're good to simplify this.